extensions/podman/packages/extension/scripts/podman-download.ts

```diff
@@ -105,31 +98,22 @@ export class PodmanDownload {
     }
     this.#shaCheck = new ShaCheck();
 
-    this.#downloadAndCheck = new DownloadAndCheck(this.#octokit, this.#shaCheck, this.#assetsFolder);
     if (!fs.existsSync(this.#assetsFolder)) {
       fs.mkdirSync(this.#assetsFolder);
     }
 
     // grab only first 2 digits from the version
     const majorMinorVersion = podmanJSON.version.split('.').slice(0, 2).join('.');
 
-    this.#podman5DownloadFedoraImage = new Podman5DownloadFedoraImage(
-      majorMinorVersion,
-      this.#octokit,
-      this.#downloadAndCheck,
-    );
-
+    const diskType = this.#platform === 'win32' ? DiskType.WSL : DiskType.Applehv;
```

Comment:
how will it work in case of Hyper-V ?

Reply:
So if is my understanding clear on windows can be diskType: `WSL`
and on Mac there can be `applehv`, `hyperV`, `qemu` ?

Reply:
mac provider can be libkrun or applehv but AFAIK it's the same disk image

and qemu is the image for Linux